### PR TITLE
Feat/database configuration

### PR DESCRIPTION
**Pull request by MajewskiOliwier**
I added basic DB config and java classes using java JPA. I also used Lombok for cleaner code. Should be good.

### CODE REVIEW

File: src/main/java/com/example/JavaProject/entity/User.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(length = 50)
+    private String userName;
+
+    private Integer age;
```
Comment (TomaszMarek) [inline src/main/java/com/example/JavaProject/entity/User.java:34]: Maybe limit that age??

File: src/main/java/com/example/JavaProject/entity/User.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+    private String email;
```
Comment (TomaszMarek) [inline src/main/java/com/example/JavaProject/entity/User.java:37]: Where is the email validation?

**Comment by TomaszMarek** [comment]
Check again which fields require a validation

**Review by TomaszMarek — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/JavaProject/entity/User.java (hunk 2 above)
Comment (MajewskiOliwier) [reply src/main/java/com/example/JavaProject/entity/User.java:37]: done

File: src/main/java/com/example/JavaProject/entity/User.java (hunk 1 above)
Comment (MajewskiOliwier) [reply src/main/java/com/example/JavaProject/entity/User.java:34]: ok, now the Top limit is 120

**Review by TomaszMarek — CHANGES_REQUESTED** [review]
Do squash!

**Review by TomaszMarek — APPROVED** [review]

**Pull request merged**